backend/FwHeadless/Program.cs

```diff
@@ -73,6 +87,9 @@ static async Task<Results<Ok<SyncResult>, NotFound, ProblemHttpResult>> ExecuteM
         return TypedResults.Ok(new SyncResult(0, 0));
     }
 
+    syncStatusService.StartSyncing(projectId);
```

Comment:
🤔 It would be pretty easy to put the number of changes that are being synced in the sync status, so that we can provide that bonus status info during a sync as well.